settings.gradle.kts

```diff
@@ -0,0 +1,23 @@
+pluginManagement {
+    repositories {
+        maven("https://gradle.pkg.st")
+    }
+}
```

Comment:
Another new feature: [Pkgst](https://docs.less.build/docs/pkgst), which caches and secures dependencies from multiple Maven repository endpoints. Pkgst modernizes download protocols, increases TLS security, and caches dependencies nearest a developer's builds via Cloudflare, with prioritized routes and several other performance/security features.

Here, we're setting the Gradle endpoint to download plugins; later in this file, we set the same for regular Maven dependencies, which allows us to use one optimized connection for all of our downloads, with caching close by to CI.

Pkgst is entirely free forever, and with Gradle's dependency locking and verification features, plus [strong TLS](https://www.ssllabs.com/ssltest/analyze.html?d=maven.pkg.st&s=104.18.14.20&hideResults=on), it is also arguably a security improvement over Central. Pkgst will fetch from Central and from [Google's Maven mirrors](https://storage-download.googleapis.com/maven-central/index.html) based on availability and best latency for the user.